### ISSUE TITLE
Cannot edit items on a layer (Woops! Someone else seems to have edited the data.)

### ISSUE THREAD

**Issue by LaurentChm** [umap-project/umap#1218]
**Describe the bug**
Within this map [https://umap.openstreetmap.fr/en/map/published-studies-on-hot-spot-detection-co2-ch4_809410#0/39/60] 
I cannot edit anymore the items on the layer "CH4 / Oil & gas" 
I could edit the items up to a couple of minutes ago but now I have a message saying _Woops! Someone else seems to have edited the data. You can save anyway, but this will erase the changes made by others_

**To Reproduce**
1. Go to https://umap.openstreetmap.fr/en/map/published-studies-on-hot-spot-detection-co2-ch4_809410#0/39/60, then layer 'CH4 / Oil & gas'
2. Edit an item
3. Save
4. See the error message

**Desktop (please complete the following information):**
 - Mac OS 12.6.4
 - Firefox 115.02

**Comment by yohanboniface** [comment]
Can you set me as an editor (ybon) of the map and allow editing for editors so I can investigate ?

**Comment by LaurentChm** [comment]
Hello Yohan, done.

**Comment by yohanboniface** [comment]
Seems to be working for me :/

![image](https://github.com/umap-project/umap/assets/146023/02009979-8bc0-4819-aa5b-d4e1d454191d)

**Comment by yohanboniface** [comment]
What if you click on "Save anyway" ?

**Comment by LaurentChm** [comment]
The red pop-up comes back again and again. Let me try again now.

**Comment by LaurentChm** [comment]
Still not working.

**Comment by yohanboniface** [comment]
Do you see my change in the layer description field (added "Test") ?

**Comment by LaurentChm** [comment]
Try to edit an item on the layer (the one next to Mongolia for ex).

**Comment by LaurentChm** [comment]
> Do you see my change in the layer description field (added "Test") ?

I don't.

**Comment by LaurentChm** [comment]
It works now. Thanks, Yohan!

**Comment by yohanboniface** [comment]
humm, I did not do anything…

**Comment by LaurentChm** [comment]
Demo effect? :)

**Comment by yohanboniface** [comment]
humm, ok, let's close issues and keep eyes open, then ;)

**Comment by LaurentChm** [comment]
And thanks for all the amazing work you and the teams behind uMap are doing!
